pkgs/min-copy-closure/liminix-rebuild.sh

```diff
@@ -20,7 +20,7 @@ if toplevel=$(nix-build "$@" -A outputs.systemConfiguration --no-out-link); then
     echo systemConfiguration $toplevel
     min-copy-closure $target_host $toplevel
     $ssh_command $target_host $toplevel/bin/install
-    $ssh_command  $target_host "sync; source /etc/profile; reboot"
+    $ssh_command  $target_host "sync; source /etc/profile; reboot -f"
```

Comment:
This is a hack but I'm not going to be  in a position to fix it properly  for a few weeks, so it only automates what i'm doing by hand anyway - fair enough.  https://gti.telent.net/dan/liminix/issues/9